src/lib/Addresses.svelte

```diff
@@ -33,13 +36,12 @@
             <tr>
                 <td class="key">RenToken</td>
                 <td class="value"
-                    ><EtherscanLink address={NETWORK.contracts.REN_TOKEN} /></td
+                    ><EtherscanLink address={config.renToken} /></td
```

Comment:
in order to avoid repeating this `<tr><td>` maybe we could pass an array of objects or an object as prop and then do a for loop like this:

```
{#each addresses as {key, value}}
           <tr>
                <td class="key">{key}</td>
                <td class="value"><EtherscanLink address={value} />
            </td>
{/each}
```

Reply:
Yeap, we can do this as well.